tsl/src/bgw_policy/job.c

```diff
@@ -371,6 +372,63 @@ policy_refresh_cagg_read_and_validate_config(Jsonb *config, PolicyContinuousAggD
 	}
 }
 
+/*
+ * Invoke compress_chunk via fmgr so that the call can be deparsed and sent to
```

Comment:
Is there a way to do this by invoking the parser on a query/SQL statement instead of doing it manually? Might be easier to adapt in the future if we'd want to, e.g., invoke compress chunks on multiple chunks in a single statement. Something like:

```
SELECT compress_chunk(ch)
FROM show_chunks(hypertable) ch LIMIT 10;
```

Reply:
I'll do the research and get back to you.

Reply:
A question here, how would this work with 2PC? Meaning, this query would generate 10 2PC transactions, correct? Would those transactions happen concurrently? Is the concurrency limited by `max_prepared_transactions`?

Reply:
I've temporarily added a `policy_invoke_compress_chunks()` function in addition to the `policy_invoke_compress_chunk()` function.

I replaced it at the access node level to see how it compares. If we are talking about doing that at the data node level then for sure it is too much work for this PR and would need to discuss it.